app/scripts/views/base.js

```diff
@@ -200,6 +200,31 @@ define(function (require, exports, module) {
         });
     },
 
+
+    /**
+     * Write HTML to the DOM
+     *
+     * @param {string || element} html
+     */
+    setHTML: function (html) {
```

Comment:
Okay, bikeshedding the method name and the argument name. I like the duality of accepting the markup or the element to inject, but calling them `setHTML` and `html` seems like it implies more the former than the latter.

How would you feel about something like along the lines of `writeToDOM` or `domInject` for the method name and a more ambiguous term like `content` for the argument? Does that make it any clearer or is the difference only inside my head?

* `writeToDOM` and `content`?
* `dom